src/models/sensors.model.ts

```diff
@@ -0,0 +1,38 @@
+import { ModelAttributes, DataTypes } from 'sequelize';
+
+export const sensorsModel:ModelAttributes = {
+    id: {
```

Comment:
id for a sensor is just a 6 bit number. When nodes send a message they will attach what data comes from which sensor id